SeaPublicWebsite/Resources/SharedResources.cy.resx

```diff
@@ -838,7 +838,7 @@
         <value>Os ydych chi’n hoffi unrhyw un o’r argymhellion hyn, gallwch eu cadw yn eich cynllun gweithredu cyn bwrw ymlaen.</value>
     </data>
     <data name="CostsAndSavingsAreEstimatesWarningTextString" xml:space="preserve">
-        <value>Amcangyfrifon yw’r costau a’r arbedion a ddarparwn ac efallai na fyddant yr un fath yn ymarferol. Mae'r arbedion a ddangosir wedi’u seilio ar gap prisiau ynni Hydref 2024 yn achos biliau nwy a thrydan, ac ar brisiau ynni RdSAP yn achos pob math arall o danwydd gwresogi. Wrth i brisiau ynni newid, felly bydd yr amcangyfrifon o arbedion.</value>
+        <value>Amcangyfrifon yw’r costau a'r arbedion rydyn ni’n eu darparu ac efallai na fyddan nhw yr un fath yn ymarferol. Mae'r arbedion sy’n cael eu dangos wedi’u seilio ar gap prisiau ynni Ionawr 2025 ar gyfer biliau nwy a thrydan, ac ar brisiau ynni RdSAP ar gyfer pob math arall o danwydd gwresogi. Wrth i brisiau ynni newid, bydd yr amcangyfrifon o’r arbedion yn newid hefyd.</value>
```

Comment:
We're not consistent with what apostrophe we use here and below: should we be? Can see it's what they've provided.

(probably this is true in general)

Reply:
have normalised this string to use ’

there are 183 ' apostrophes, and 603 ’ apostrophes. can make a ticket to normalise this? probably worth raising to DESNZ if it's noticeable if there's anything they can do on their end

Reply:
If it's noticeable on the site (?) then I think we should raise a ticket to resolve. Otherwise don't think it's worth the effort.